packages/wp-now/README.md

```diff
@@ -89,211 +91,234 @@ cd wordpress-plugin-or-theme
+
+As the building blocks of Playground, Blueprints offer advanced functionality, and we recommend you [learn more about how to use them](https://wordpress.github.io/wordpress-playground/blueprints-api/index) before you get started.
+
+Here's an example of a Blueprint that defines a custom URL constant in `wp-config.php` using [`WP_HOME`](https://developer.wordpress.org/advanced-administration/wordpress/wp-config/#blog-address-url) and [`WP_SITEURL`](https://developer.wordpress.org/advanced-administration/wordpress/wp-config/#wp-siteurl).
+
+Assuming you added `myurl.wpnow` to your `hosts` file, your site will be available at `http://myurl.wpnow`.
+
+Note that the `method` is set to `define-before-run` to avoid modifying the shared `wp-config.php` file. The default method, `rewrite-wp-config`, _modifies_ the `wp-config.php` on the disk.
 
 ```json
 {
-	"steps": [
-		{
-			"step": "defineWpConfigConsts",
-			"consts": {
-				"WP_HOME": "http://myurl.wpnow",
-				"WP_SITEURL": "http://myurl.wpnow"
-			},
-			"method": "define-before-run"
-		}
-	]
+  "steps": [
+    {
+      "step": "defineWpConfigConsts",
+      "consts": {
+        "WP_HOME": "http://myurl.wpnow:8881",
+        "WP_SITEURL": "http://myurl.wpnow:8881"
+      },
+      "method": "define-before-run"
+    }
+  ]
 }
 ```
 
-### Defining Debugging Constants
+You can use this instance with [`ngrok`](https://ngrok.com/docs):
+1. Run `ngrok http 8881`
+2. Copy the URL, and
+3. Replace `WP_HOME` and `WP_SITEURL` in the Blueprint file.
 
-In the similar way we can define `WP_DEBUG` constants and read the debug logs.
+You can also define a different port:
 
-Run `wp-now start --blueprint=path/to/blueprint-example.json` where `blueprint-example.json` is:
+```bash
+wp-now start --blueprint=path/to/blueprint-example.json --port=80
+```
+
+The Blueprint to listen on port `80` would look like this:
 
 ```json
 {
-	"steps": [
-		{
-			"step": "defineWpConfigConsts",
-			"consts": {
-				"WP_DEBUG": true
-			}
-		}
-	]
+  "steps": [
+    {
+      "step": "defineWpConfigConsts",
+      "consts": {
+        "WP_HOME": "http://myurl.wpnow",
+        "WP_SITEURL": "http://myurl.wpnow"
+      },
+      "method": "define-before-run"
+    }
+  ]
 }
 ```
 
-Because we didn't define a method for `defineWpConfigConsts`, and the default method is `rewrite-wp-config`, this blueprint will update your `~/.wp-now/wordpress-versions/latest/wp-config.php` enabling the WP_DEBUG constant and will create a `debug.log` file in the `~/.wp-now/wp-content/${project}/debug.log` directory.
-The next time you execute `wp-now start` in any project, the variable `WP_DEBUG` will still be set to true.
+### Define debugging constants in a Blueprint
 
-If you prefer to set a custom path for the debug log file, you can set `WP_DEBUG_LOG` to be a directory. Remember that the `php-wasm` server runs under a VFS (virtual file system) where the default documentRoot for wp-now is always `/var/www/html`.
```

Comment:
@flexseth, BTW, that's covered above (see [line 133 onward](https://github.com/WordPress/playground-tools/blob/e6c9f13571db056d21d353cac4ceadcaf7704e77/packages/wp-now/README.md?plain=1#L133-L139))

Reply:
just finally seeing this in the docs for the first time 🤔

Something about the organization of this page is a little weird for me. It's probably just because it covers a ton of topics. For instance in the DevHub, there's the sidebar at the right of the page with quick jump links to sections on the page.. this one just has a weird flow. But it's what we have for now :)

Reply:
Yeah, the Devhub is a full on docs site; this is just a README 😄 

I think I've mentioned it here somewhere, but there's a new chapter in the docs dedicated to local development (it covers `wp-now` and the VS Code extension).
You're welcome to suggest a reorg/additions to it in the main repo!

Reply:
> @flexseth, BTW, that's covered above (see [line 133 onward](https://github.com/WordPress/playground-tools/blob/e6c9f13571db056d21d353cac4ceadcaf7704e77/packages/wp-now/README.md?plain=1#L133-L139))

ah, I see. 

### Visualization
There's something about the structure of the page here that throws me off... maybe add some more to the table-of-contents about the filesystem?

Maybe like a flowchart or diagram that shows where files go depending on the boot mode? 

I've found the filesystem part of `wp-now` to be a bit confusing and have been fiddling with it a good bit. Maybe can come up with some flows... stay tuned